src/highdicom/seg/sop.py

```diff
@@ -581,8 +582,13 @@ def add_segments(
                 # If Dimension Organization Type is TILED_FULL, plane
                 # positions are implicit and need to be computed.
                 image_origin = src_img.TotalPixelMatrixOriginSequence[0]
-                orientation = tuple(
-                    float(v) for v in src_img.ImageOrientationSlide
+                orientation = (
```

Comment:
If you don't want to write this out explicitly, you could replace the type annotation on the function def with `Tuple[float, ...]`, although of course this would be less specific

Reply:
Yes, I wanted that to be specific because the number of items matter in this case.